app/temporal/src/workflows/importer.workflow.ts

```diff
@@ -173,21 +173,22 @@ export async function importer(params: ImporterWorkflowParams) {
         const columnValidations = columnConfig.validations;
 
         if (columnValidations?.length) {
+          const validationResults = await performRecordValidation(
```

Comment:
ich glaube das ist ziemlich inperformant, da du bei unique columns jetzt alle validations doppelt laufen lässt.

Reply:
Ist aber als Workaround notwendig, damit ich die Messages bekomme. Wir können das in #68 verbessern. Aktuell läuft es eigentlich ganz gut durch